crowbar_framework/lib/crowbar/upgrade.rb

```diff
@@ -43,14 +43,15 @@ def knife_files
 
       crowbar_databags_path = @data.join("knife", "databags", "crowbar")
       crowbar_databags_path.children.each do |file|
-        if file == "bc-template-nova_dashboard.json"
-          crowbar_databags_path.join(file).rename(
-            crowbar_databags_path.join(file.sub!("nova_dashboard", "horizon"))
-          )
-
+        if file.to_s.match("bc-nova_dashboard-default.json$")
```

Comment:
Performance/RedundantMatch: Use `=~` in places where the `MatchData` returned by `#match` will not be used.